Gemeinsamer Bibliotheksverbund ISBN.js

```diff
@@ -0,0 +1,238 @@
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "import",
+		"input": "{ \"ISBN\" : \"3-86688-240-8\" }",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Bilinguale Lexik nicht materieller lexikalischer Transfer als Folge der aktuellen russisch-deutschen Zweisprachigkeit",
+				"creators": [
+					{
+						"firstName": "Katrin",
+						"lastName": "Karl",
+						"creatorType": "author"
+					}
+				],
+				"date": "2012",
+				"ISBN": "9783866882416",
+				"language": "ger",
+				"numPages": "683",
+				"place": "München",
+				"publisher": "Verl. Otto Sagner",
+				"series": "Slavolinguistica",
+				"seriesNumber": "15",
+				"attachments": [
+					{
+						"title": "Table of Contents PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [
+					"CD-ROM",
+					"CD-ROM",
+					"Deutsch",
+					"Deutsch",
+					"Deutsch",
+					"Online-Publikation",
+					"Online-Publikation",
+					"Russisch",
+					"Russisch",
+					"Russisch",
+					"Wortschatz",
+					"Wortschatz",
+					"Wortschatz",
+					"Zweisprachigkeit",
+					"Zweisprachigkeit",
+					"Zweisprachigkeit"
+				],
+				"notes": [
+					{
+						"note": "Univ., FB SLM, Diss. 2011 u.d.T.: Bente Karl, Katrin: Nicht materieller lexikalischer Transfer als Folge der aktuellen russisch-deutschen Zweisprachigkeit--Hamburg, 2011"
+					},
+					{
+						"note": "Was ist das Besondere an bilingualer Lexik? Während Entlehnungen und Code-Switching in der bisherigen Forschung breit untersucht wurden, widmet sich die vorliegende Arbeit einem Desiderat und untersucht ausführlich die theoretische und empirische Natur nicht materieller lexikalischer Transfers (u.a. auch bekannt als Calques) bezogen auf die aktuelle Kontaktsituation des Russischen mit dem Deutschen. Im Theorieteil der Arbeit erfolgt eine Abgrenzung möglicher Kontaktphänomene und eine Subklassifizierung der nicht materiellen lexikalischen Transfers in insgesamt fünf Kategorien. Im empirischen Teil werden ein umfangreiches bilinguales Korpus, bestehend aus spontaner mündlicher Kommunikation sowie schriftlicher Sprachproduktion, ausgewertet und Ergebnisse einer computerunterstützt durchgeführten Bewertungsaufgabe präsentiert. Wo in dem einen Teil die realen Vorkommen im Vordergrund stehen, ist es in dem anderen Teil die Bewertung von nicht materiellen lexikalischen Transfers wie otkladyvat' na dlinnuju skamejku 'auf die lange Bank schieben`, wobei die Urteile Bilingualer mit denjenigen einer monolingualen Kontrollgruppe verglichen werden"
+					}
```

Comment:
Two notes come from MARC fields 500 and 520 (I didn't changed that). However, we want to move the 520 to abstract, which will leave the 500 note (general note) alone. Is that okay?